crates/extensions/c8y_http_proxy/src/actor.rs

```diff
@@ -57,9 +58,9 @@ impl Actor for C8YHttpConfig {
         "C8YHttpProxy"
     }
 
-    async fn run(self, messages: Self::MessageBox) -> Result<(), ChannelError> {
+    async fn run(self, messages: Self::MessageBox) -> Result<(), RuntimeError> {
         let actor = C8YHttpProxyActor::new(self, messages);
-        actor.run().await
+        Ok(actor.run().await?)
```

Comment:
That looks weird. Wondering why the `C8YHttpProxyActor::run` is not returning `RuntimeError`

Reply:
Good catch. This has been fixed by https://github.com/thin-edge/thin-edge.io/pull/1752/commits/f9fd119c2ce26bbf19756a4c8f88e0b418ad1a01